ui/apps/platform/src/Containers/ComplianceEnhanced/Status/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -66,9 +69,17 @@ function ScanResultsOverviewTable() {
         }
 
         return scanResultsOverviewData?.map(({ scanStats, clusterId, profileName }) => (
-            // TODO: verify scanName unique
-            <Tr key={scanStats.scanName}>
-                <Td>{scanStats.scanName}</Td>
+            <Tr key={scanStats.id}>
+                <Td>
+                    <Button
+                        variant={ButtonVariant.link}
+                        isInline
+                        component={LinkShim}
+                        href={generatePath(complianceEnhancedStatusScansPath, { id: scanStats.id })}
+                    >
+                        {scanStats.scanName}
+                    </Button>
```

Comment:
Any downside in just rendering a `<Link>` instead of the link-styled-button? I know for VM 2.0 we are probably replacing this pattern as it makes it so the user cannot copy/paste the link text.

Reply:
Good callout, I already forgot about that conversation 😅